tests/scripts/run-test-suites.pl

```diff
@@ -35,8 +35,9 @@
 
 # All test suites = executable files, excluding source files, debug
 # and profiling information, etc. We can't just grep {! /\./} because
-#some of our test cases' base names contain a dot.
+# some of our test cases' base names contain a dot.
 my @suites = grep { -x $_ || /\.exe$/ } glob 'test_suite_*';
```

Comment:
On OS X, when running `basic-build-test.sh` this script is picking up the `.dSYM` directory, so this expression should be changed to glob only files. We should have:
```
@suites = grep { !/\.c$/ && !/\.data$/ && -f } @suites;
```